.circleci/config.yml

```diff
@@ -108,298 +95,82 @@ jobs:
       - run:
           name: Verify format
           command: yarn format:check
-      - run:
-          name: Verify lint
-          command: yarn lint:check
       - run:
           name: 'lint package.json files'
           command: 'yarn run packageJsonLint'
-  test-request-client:
+  test-unit:
     docker:
       - *node_image
-      - *ganache_image
+    resource_class: xlarge
     working_directory: *working_directory
     steps:
       - attach_workspace:
           at: *working_directory
       - run:
-          name: 'Build request-client.js'
-          command: 'yarn workspace @requestnetwork/request-client.js run build:commonjs'
-      - run:
-          name: 'Deploy test contracts from smart-contracts'
-          command: 'yarn workspace @requestnetwork/smart-contracts run deploy'
-      - run:
-          name: 'Test request-client.js'
-          command: 'yarn workspace @requestnetwork/request-client.js run test --ci --maxWorkers=1'
+          name: 'Test unit'
+          command: 'yarn test --since master --ignore @requestnetwork/request-client.js --ignore @requestnetwork/smart-contracts --ignore @requestnetwork/payment-detection --ignore @requestnetwork/payment-processor --ignore @requestnetwork/integration-test'
```

Comment:
I see a minor disadvantage: if a test fails, it's easier to retry the specific test when they're divided into parallel jobs. Other than that, looks good to me!